jscomp/core/lam_compile.mli

```diff
@@ -25,6 +25,6 @@
 (** Compile single lambda IR to JS IR  *)
 
 val compile_recursive_lets :
-  Lam_compile_context.t -> (Ident.t * Lam.t) list -> Js_output.t
+  output_prefix:string -> Js_packages_info.module_system -> Lam_compile_context.t -> (Ident.t * Lam.t) list -> Js_output.t
```

Comment:
About invasive:

This and the function below are the only 2 entry points where the 2 new parameters are passed in.
No way to thread them though each recursive call if they never change and are passed from the outside.

Reply:
Once the number of changes is reduced, it should be possible to just isolate what change is responsible.
E.g. perhaps first remove Pimport, then remove gradually every other change in turn, locally, until the tests stops failing an one finds out the root cause.